app/routes/_auth+/signup.tsx

```diff
@@ -156,10 +160,10 @@ export default function SignupRoute() {
 	const signupFetcher = useFetcher<typeof action>()
 	const [form, fields] = useForm({
 		id: 'signup-form',
-		constraint: getFieldsetConstraint(createSchema()),
+		constraint: getFieldsetConstraint(emailFieldSchema),
 		lastSubmission: signupFetcher.data?.submission,
 		onValidate({ formData }) {
-			return parse(formData, { schema: createSchema() })
+			return parse(formData, { schema: emailFieldSchema })
```

Comment:
@kentcdodds 

without this change `onboarding.test.ts` was failed, I did not find in `conform-to/react` a way to use asunc `createSchema` here.

the test work and i've add a test for `user already exists` case